metron-interface/metron-alerts/package.json

```diff
@@ -5,12 +5,14 @@
   "angular-cli": {},
   "scripts": {
     "build": "./node_modules/@angular/cli/bin/ng build --prod",
-    "start": "ng serve",
+    "start": "ng serve --aot",
```

Comment:
why is `--aot` needed? It slows down the compiler in development mode.

I'm assuming you wanted to be consistent with the [start-dev](https://github.com/apache/metron/blob/master/metron-interface/metron-alerts/scripts/start-dev.sh#L19) script, right?

Reply:
Thank you for this comment @ruffle1986 - this is the kind of detail we need community help from UI devs in reviewing.

Reply:
Thanks @ruffle1986 for checking this! We've got the ahead of time compilation turned on in our start scripts too. It's needed because of the lazy loading of the PCAP UI.